tests/safeds/data/tabular/containers/_table/test_transform_table.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.exceptions import TransformerNotFittedError, UnknownColumnNameError
+from safeds.data.tabular.transformation import OneHotEncoder
+
+
+class TestTransform:
```

Comment:
Also include the tests for `OneHotEncoder.fit_and_transform` here. These check
* that the transformed table is returned,
* that the original table is not changed.

Naturally, you need to call `fit` separately first.